### DISCUSSION TITLE
PnP Search query based on date

### DISCUSSION THREAD

**Discussion by madslmoeller** [microsoft-search/pnp-modern-search#4156] [Q&A]
I have a list which I am using as an alert list through PnP based on specific dates. Until now I worked with LastModifiedTime="this week", but request is that it should be based on the StartDate and EndDate columns now.

{searchTerms} path:https://xxx.sharepoint.com/sites/Intranet/Lists/Alerts
AND * AND ID:*
AND StartDate<=[Today]
AND EndDate>=[Today]

No elements appearing..

**Comment by wobba** [comment]
Take a look at the docs at https://microsoft-search.github.io/pnp-modern-search/usage/search-results/tokens/#date-tokens
